cmd/qat_plugin/qat_plugin_test.go

```diff
@@ -31,20 +31,36 @@ func init() {
 	debug.Activate()
 }
 
-func createTestFiles(prefix string, dirs []string, files map[string][]byte) error {
+func createTestFiles(prefix string, dirs []string, files map[string][]byte, symlinks map[string]string) error {
 	for _, dir := range dirs {
+
 		err := os.MkdirAll(path.Join(prefix, dir), 0755)
 		if err != nil {
 			return errors.Wrap(err, "Failed to create fake device directory")
 		}
+
 	}
 	for filename, body := range files {
+
 		err := ioutil.WriteFile(path.Join(prefix, filename), body, 0644)
 		if err != nil {
 			return errors.Wrap(err, "Failed to create fake vendor file")
 		}
+
 	}
+	for link, target := range symlinks {
 
+		if target != "" {
```

Comment:
Do you think we can handle this in a better way? I am adding this to handle this testcase here https://github.com/intel/intel-device-plugins-for-kubernetes/pull/79/files#diff-e22a18bb5131c8f3818190890b51c4a1R331 where the target is empty.

Reply:
Good question.

IIUC the value of `link` is not used at all if `target` is empty. Then why having `symlinks` in this test case in first place? I guess it'd be easier not to set `symlinks` in the test case and to remove this `if target != ""` check from the function. I'm OK with setting `symlinks` for one test case only (the one called `vfio-pci DPDKdriver with  one kernel bound device (QAT device) where vfdevID is equal to qatDevId (37c9)`).

Reply:
Okay, removing the symlink from the testcase and removing the check for empty string at the time of symlink creation.